catroid/src/main/java/org/catrobat/catroid/io/BackpackSerializer.java

```diff
@@ -1,6 +1,6 @@
 /*
  * Catroid: An on-device visual programming system for Android devices
- * Copyright (C) 2010-2018 The Catrobat Team
+ * Copyright (C) 2010-2021 The Catrobat Team
```

Comment:
Please undo the changes on this file as there is just a change of formatting.

Reply:
Actually to reduce code redundancy I had modified BackPackSerializer so that I could use the same class to serialize both the backpack as well as the Recent Bricks, to achieve this I modified the BackpackSerializer constructor to accept a "File" object which will be used to save the serialized file for either use case. In this way, the same serializer could be used to save RecentBricksFile for Recent Bricks and Backpack File for Backpack. I had created the RECENT_BRICKS_DIRECTORY name for the same purpose but forgot to use it in code, I will do the necessary and push the code again? although I could make a different serializer for the Recent Bricks but I don't think that would make much sense, considering it will contain the same code?

Reply:
Ok that is fine then! No need to produce duplicate code.